pkg/crypto/keys/nep2.go

```diff
@@ -52,10 +53,15 @@ func NEP2Encrypt(priv *PrivateKey, passphrase string, params ScryptParams) (s st
 	if err != nil {
 		return s, err
 	}
+	defer slice.Clean(derivedKey)
 
 	derivedKey1 := derivedKey[:32]
 	derivedKey2 := derivedKey[32:]
-	xr := xor(priv.Bytes(), derivedKey1)
+
+	privBytes := priv.Bytes()
+	defer slice.Clean(privBytes)
+	xr := xor(privBytes, derivedKey1)
+	defer slice.Clean(xr)
```

Comment:
It's just an intermediate result, why should we clean it?

Reply:
It still leaks some data, even though it's masked. "paranoia: on" mode.